src/mongocxx/test/index_view.cpp

```diff
@@ -344,9 +344,9 @@ TEST_CASE("drop_all", "[index_view]") {
         bsoncxx::document::view result_view = result.view();
 
         auto cursor1 = indexes.list();
-        REQUIRE(std::distance(cursor1.begin(), cursor1.end()) == models.size() + 1);
-        REQUIRE((result_view["numIndexesAfter"].get_int32() -
-                 result_view["numIndexesBefore"].get_int32()) == models.size());
+        REQUIRE((unsigned)std::distance(cursor1.begin(), cursor1.end()) == models.size() + 1);
+        REQUIRE((unsigned)(result_view["numIndexesAfter"].get_int32() -
```

Comment:
I'm actually not sure why this wasn't caught earlier. This is a comparison of a signed an unsigned int, which throws a warning, which we mark as an error.

Reply:
Huh, it may be that we're not using a new enough version of g++ in evergreen. The tests on Ubuntu 16.04 use g++ 5.4.0. This is an improvement regardless.